api/api.pb.txt

```diff
@@ -711,6 +711,13 @@ file {
       json_name: "swiftPrefix"
     }
     field {
+      name: "php_class_prefix"
```

Comment:
interesting, I see these were removed in a previous commit; https://github.com/docker/swarmkit/commit/8f9f7dc1c1e1bdf21a7c31d4affbd1f74298060c#diff-832df52f65be6ede52a44115a7cfe9e5L714

different version used to generate?

Reply:
Rebase? I think this fix was merged.

Reply:
Yes that fix was, but.. https://github.com/docker/swarmkit/pull/2451 😅